ci/check-protobuf-stability.sh

```diff
@@ -3,7 +3,18 @@
 # SPDX-License-Identifier: Apache-2.0
 set -euo pipefail
 
-# LF protobufs are checked against local snapshosts.
+eval "$(dev-env/bin/dade assist)"
```

Comment:
Since you added this globally, you may drop the one at line 20/31, 
meaning replace 
```
  (eval "$(dev-env/bin/dade assist)"; buf breaking --config "buf-lf-transaction.yaml" --against ${stable_snapshot_dir})
```
by 
```
  buf breaking --config "buf-lf-transaction.yaml" --against ${stable_snapshot_dir}
```

Reply:
Good point, I can remember we had trouble with enabling this globally, I'll try to find the relevant discussion.

Reply:
https://github.com/digital-asset/daml/pull/10950#pullrequestreview-760652273

Reply:
You should not enable it globally if you plan to checkout old commits, in which dev-env might have changed.